scripts/setup_ubuntu.sh

```diff
@@ -85,15 +91,18 @@ run_as_original_user pip3 install --upgrade "pip$PIP_VERSION" "setuptools$SETUPT
 run_as_original_user pip3 install conan==1.36 PyGithub packaging
 
 # Developer tools
-DEVELOPER_TOOLS="clang-format-10 clang-tidy-10 doxygen graphviz ccache cmake shellcheck"
+#
+# pkg-config is required to build pyarrow correctly (seems to be a bug)
+DEVELOPER_TOOLS="clang-format-10 clang-tidy-10 doxygen graphviz ccache cmake shellcheck pkg-config"
 # github actions require a more recent git
 GIT=git
 # Library dependencies
 #
 # Install llvm via apt instead of as a conan package because existing
 # conan packages do yet enable RTTI, which is required for boost
 # serialization.
-LIBRARIES="libxml2-dev llvm-10-dev"
+LIBRARIES="libxml2-dev llvm-10-dev libarrow-dev=4.0.1-1 libarrow-python-dev=4.0.1-1 libparquet-dev=4.0.1-1
```

Comment:
minor: we have much longer lines, why break here

Reply:
I'll break the other lines. :-)